test/export/StructureDefinitionExporter.test.ts

```diff
@@ -153,4 +155,82 @@ describe('StructureDefinitionExporter', () => {
     expect(changedCard.min).toBe(1);
     expect(changedCard.max).toBe('1');
   });
+
+  // Flag Rule
+  it('should apply a valid flag rule', () => {
+    const profile = new Profile('Foo');
+    profile.parent = 'DiagnosticReport';
+
+    const rule = new FlagRule('conclusion');
+    rule.modifier = false;
+    rule.mustSupport = true;
+    profile.rules.push(rule);
+
+    const sd = exporter.exportStructDef(profile, input);
+    const baseStructDef = sd.getBaseStructureDefinition();
+
+    const baseElement = baseStructDef.findElement('DiagnosticReport.conclusion');
+    const changedElement = sd.findElement('DiagnosticReport.conclusion');
+    expect(baseElement.isModifier).toBeFalsy();
+    expect(baseElement.mustSupport).toBeFalsy();
+    expect(baseElement.isSummary).toBeFalsy();
+    expect(changedElement.isModifier).toBe(false);
+    expect(changedElement.mustSupport).toBe(true);
+    expect(baseElement.isSummary).toBeFalsy();
+  });
+
+  it('should not apply a flag rule that disables isModifier', () => {
+    const profile = new Profile('Foo');
+    profile.parent = 'DiagnosticReport';
+
+    const rule = new FlagRule('status');
+    rule.modifier = false;
+    rule.mustSupport = true;
+    profile.rules.push(rule);
+
+    const sd = exporter.exportStructDef(profile, input);
+    const baseStructDef = sd.getBaseStructureDefinition();
+
+    const baseElement = baseStructDef.findElement('DiagnosticReport.status');
+    const changedElement = sd.findElement('DiagnosticReport.status');
+    expect(baseElement.isModifier).toBe(true);
+    expect(baseElement.mustSupport).toBeFalsy();
+    expect(changedElement.isModifier).toBe(true);
+    expect(changedElement.mustSupport).toBeFalsy();
+  });
+
+  it('should not apply a flag rule that disables mustSupport', () => {
+    const jsonMustSupport = JSON.parse(
+      fs.readFileSync(
+        path.join(__dirname, '../fhirdefs/testdefs/patient-photo-must-support.json'),
+        'utf-8'
+      )
+    );
+    const mustSupportDefs: FHIRDefinitions = new FHIRDefinitions('4.0.1');
+    mustSupportDefs.add(jsonMustSupport);
+    const mustSupportExporter: StructureDefinitionExporter = new StructureDefinitionExporter(
+      mustSupportDefs
+    );
```

Comment:
The above is clever, but actually unnecessary.  We have access to any of the profiles defined as part of core FHIR ([listed here](http://hl7.org/fhir/R4/profilelist.html)).  So, you could skip all this and just create a profile that declares its parent as `http://hl7.org/fhir/StructureDefinition/vitalsigns` (which has lots of MS, as [shown here](http://hl7.org/fhir/R4/vitalsigns.html)).

Reply:
That's

that's good. That's much better. I'll do that. I am extremely embarrassed by this.

Reply:
Ha.  Nothing to be embarrassed by at all.  I don't think I _told_ anyone we supported those other FHIR profiles until now.  So really, I shouldn't have expected you to have known!